packages/rocketchat-contacts/server/startup.js

```diff
@@ -22,4 +22,19 @@ RocketChat.settings.addGroup('Contacts', function() {
 		public: true,
 		i18nDescription: 'Contacts_Background_Sync_Interval_Description',
 	});
+	this.add('Contacts_Dynamic_Link_APIKey', '', {
```

Comment:
I don't see where these three values are being set..... the i18n file doesn't seem to have them?

Reply:
The admin console under the section 'Contacts'